install-k8s.yml

```diff
@@ -6,8 +6,17 @@
     - runtime
     - download-k8s
     - install-k8s
+    - role: install-cni-plugins
+      when: ansible_architecture =="s390x"
 
 - name: Install networking - calico
   hosts: masters
   roles:
-    - install-calico
+    - role: install-calico
+      when: ansible_architecture == "ppc64le"
```

Comment:
instead of choosing arch wise, we need to find a way to choose cni type via group args because there is a possibility that you may install flannel on ppc64le as well

Reply:
Yes added cni_provider variable to handle this